metadata.json

```diff
@@ -27,12 +27,16 @@
     {
       "name": "puppetlabs/stdlib",
       "version_requirement": ">= 4.13.1 < 6.0.0"
+    },
+    {
+      "name": "puppetlabs/selinux_core",
```

Comment:
I think this is wrong. The module is only marked as puppet >= 6 compatible.

Reply:
hm. It is a dependency though :/ Puppet's metadata format is not expressive enough in this case.

I think in order to maintain backwards compatibility, we'll just need to note in the readme that selinux_core is needed for Puppet >= 6.0.0

Reply:
That *shouldn't* be necessary.  If your puppet-agent doesn't automatically come with it, something has gone wrong.